docs/includes/enable-debugging-on-edge-devtools.md

```diff
@@ -0,0 +1,38 @@
+When the add-in is running in Microsoft Edge, UI-less code will not be able to attach to a debugger by default.
```

Comment:
This sentence parses strangely!
Consider "If you run the add-in in the Edge browser, UI-less code will not be able to attach to a debugger by default."